pageserver/src/tenant/size.rs

```diff
@@ -267,7 +379,9 @@ pub(super) async fn gather_inputs(
     // for branch points, which come as multiple updates at the same LSN, the Command::Update
     // is needed before a branch is made out of that branch Command::BranchFrom. this is
     // handled by the variant order in `Command`.
+    //
```

Comment:
The whole above comment is now outdated.

Reply:
not really, we still preserve this Update < BranchFrom invariant

Reply:
Aah yes, that still holds.